Sources/StreamVideo/WebSockets/Client/BackgroundTaskScheduler.swift

```diff
@@ -36,7 +38,7 @@ class IOSBackgroundTaskScheduler: BackgroundTaskScheduler {
     var isAppActive: Bool {
         let app = self.app
         if Thread.isMainThread {
-            return app?.applicationState == .active
+            return app?.applicationState == .active || callKitService.callCount > 0
```

Comment:
is it possible to move this logic somewhere higher up? Long term, we would like to use the same WS implementation and bg task schedulers across all SDKs, and this would be an obstacle.